test/e2e/marketplace_test.go

```diff
@@ -1,41 +1,67 @@
 package e2e
```

Comment:
Does it make sense that, since this file is also now a harness, that `main_test.go` is no longer a useful abstraction and we should just move the `TestMain(m *testing.M)` function into `marketplace_test.go` ?

Reply:
I'd support this change.